example/lib/database.dart

```diff
@@ -1,13 +1,16 @@
 import 'dart:async';
 
+import 'package:example/date_time_converter.dart';
 import 'package:example/task.dart';
 import 'package:example/task_dao.dart';
+import 'package:example/timestamp.dart';
 import 'package:floor/floor.dart';
 import 'package:sqflite/sqflite.dart' as sqflite;
 
 part 'database.g.dart';
 
-@Database(version: 1, entities: [Task])
+@TypeConverters([DateTimeConverter])
+@Database(version: 1, entities: [Task], embeds: [Timestamp])
```

Comment:
Do we need to include the embeds here? doesn't it suffice to annotate the usage in the entities/views?

Reply:
We need to include if we annotate the embedded class, if we annotate the variable like room then we don't need to include here.